src/main/java/com/iexec/sms/secret/app/ApplicationRuntimeSecretRepository.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.sms.secret.app;
+
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.Optional;
+
+public interface ApplicationRuntimeSecretRepository extends CrudRepository<ApplicationRuntimeSecret, String> {
+    Optional<ApplicationRuntimeSecret> findByAddressIgnoreCaseAndIndex(String address, long index);
```

Comment:
You're right. I copied what's already done in [Web3](https://github.com/iExecBlockchainComputing/iexec-sms/blob/develop/src/main/java/com/iexec/sms/secret/web3/Web3SecretService.java) and [Web2](https://github.com/iExecBlockchainComputing/iexec-sms/blob/develop/src/main/java/com/iexec/sms/secret/web2/Web2SecretsService.java) secrets services.
The only difference is that I use `findByAddressIgnoreCase` instead of use `.lowerCase` before finding by address. Do you think it could be an issue?

Reply:
findByAddressIgnoreCase() on other parts ensures compatibility with older versions where addresses were not lowercased, but this feature is new, so we don't need findByAddressIgnoreCase()

Reply:
Right, thanks!